movies/models.py

```diff
@@ -23,4 +24,18 @@ def __str__(self):
 
     @cached_property
     def get_imdb_url(self):
-        return f'{settings.IMDB_URL}{self.imdb_id}'
+        return f'{base_settings.IMDB_URL}{self.imdb_id}'
+
+    def save(self, *args, **kwargs):
```

Comment:
It looks good. I see you are still creating thumbnails on each save, which is not the optimal solution, but we can continue, no problem. You can read this to get an idea of how this could be implemented: https://stackoverflow.com/questions/36719566/identify-the-changed-fields-in-django-post-save-signal

Reply:
Thank you!